tests/__mocks__/mocks.ts

```diff
@@ -1,16 +1,16 @@
 import type { WriteStream } from 'node:fs'
 import { type HookStats, type RunnerStats, type SuiteStats, type Test, TestStats } from '@wdio/reporter'
-import type { Capabilities } from '@wdio/types'
 import type {
     ConfigCapabilities,
     DesiredCapabilitiesExtended,
     HookStatsExtended,
     RunnerStatsExtended,
     SuiteStatsExtended,
     TestStatsExtended,
-    TestrunnerExtended,
+    TestrunnerExtended
 } from '../../src/types/wdio'
 import type { Feature, Scenario, cjson_metadata } from '../../src/types'
+import { PageLoadingStrategy } from '@wdio/types/build/Capabilities'
```

Comment:
Let's not import from a package path as that can change, better from the package directly and then do `Capabilities.PageLoadingStrategy`

```suggestion
import { Capabilities } from '@wdio/types'
```

Reply:
Sure, updated. Thanks!